week-3/extended-project/champion-performance/model/team_members.json

```diff
@@ -0,0 +1,32 @@
+{
+  "paul": {
+    "name": "Paul Portillo",
+    "photo": "https://avatars3.githubusercontent.com/u/5809281?s=400&u=fb6ec2c13e7df3d919f40cf46c14b9360de6cd66&v=4",
+    "team": "Champion Performance",
+    "introduction": "Hi i'm Ricardo, I like to drink coffee at the afternoon because i'm very sleepy after lunch, I love team work because we can do big things together, but sometimes I like to focus on my work, I enter the zone!, an the signal of that is that i put my headphone on, so on those times please let me focus. I like feedback, but i like it more when there is a reasoning behind it."
```

Comment:
You gotta pay attention to your content (introduction). Not only here for "paul", but also for "paulo".

Reply:
😞